docs/devGuide/devGuide.md

```diff
@@ -80,6 +80,22 @@ You can find more information at the [VueStrap repo](https://github.com/MarkBind
 
 ## Development Process
 
+### Prerequisite
+
+MarkBind requires the following dependencies
+
+* Node.js (LTS or higher) [with npm version of 6.12.1 or higher]
+* Java 8 or later (required to run the PlantUML JAR executable)
+* Graphviz v2.38 or later (required to generate all diagrams)
```

Comment:
Nit: Generate "PlantUML diagrams", rather than "all diagrams"? In general, we try to be as specific as possible.